front/pages/api/w/[wId]/vaults/[vId]/index.ts

```diff
@@ -72,22 +75,21 @@ async function handler(
         view.toJSON()
       );
 
-      const categories = serializedDatasourceViews.reduce(
-        (acc, dataSourceView) => {
-          const value = acc[dataSourceView.category];
-          if (value) {
-            value.count += 1;
-            value.usage += dataSourceView.usage;
-          } else {
-            acc[dataSourceView.category] = {
-              count: 1,
-              usage: dataSourceView.usage,
-            };
-          }
-          return acc;
-        },
-        {} as { [key: string]: VaultCategoryInfo }
-      );
+      const categories: { [key: string]: VaultCategoryInfo } = {};
+      DATA_SOURCE_VIEW_CATEGORIES.forEach((category) => {
```

Comment:
You don't like reduce 😛?

Reply:
I want all categories displayed even if no data inside it 😅